### DISCUSSION TITLE
Welcome new committer:Li Zhi Qiang

### DISCUSSION THREAD

**Discussion by seeflood** [mosn/layotto#414] [Announcements]
Hi Community,

On behalf of the Layotto PMC, I am pleased to announce that Li Zhi Qiang a.k.a stulzq, has been voted in as Layotto committer.

![image](https://user-images.githubusercontent.com/26001097/158166759-89ddf621-5ad9-4001-838d-1e4d4abdcbf2.png)

Li Zhi Qiang has been making great contributions to Layotto. Please join me to say congratulations to him!

@stulzq  , would you please briefly introduce yourself to the community?

Best Regards.

**Comment by zhenjunMa** [comment]
welcome! 👍

**Comment by stulzq** [comment]
Hello everyone, I'm Li Zhi Qiang, open source enthusiasts, currently working on the ly.com. I am very happy to receive this honor. I have learned a lot in layotto. Thank the community for the opportunity.

**Reply by nobodyiam** [reply]
Welcome!
